src/transformers/cache_utils.py

```diff
@@ -1957,16 +1988,27 @@ def __init__(
         config: PretrainedConfig,
         max_batch_size: int,
         max_cache_len: Optional[int],
-        device: Union[str, torch.device],
+        device: Union[str, torch.device] = None,
         dtype: Optional[torch.dtype] = None,
         offload_device: Union[str, torch.device] = torch.device("cpu"),
         layer_device_map: Optional[Dict[int, Union[str, torch.device, int]]] = None,
+        **kwargs,
     ) -> None:
+        Cache.__init__(self)
```

Comment:
```suggestion
        super(Cache, self).__init__()
```
(I think this is the right syntax?)